frame/staking/src/pallet/mod.rs

```diff
@@ -439,6 +476,61 @@ pub mod pallet {
 		ValueQuery,
 	>;
 
+	/// The nominator count each `ExposurePage` is capped at.
+	///
+	/// This cannot be greater than `T::MaxNominatorRewardedPerValidator`.
```

Comment:
Needs to have an integrity check that makes sure the default value of this is always less than the bound. And it should go into `try_state` as well. 

See: https://github.com/paritytech/substrate/pull/13120/files#diff-103bae13a15813f338cb0bb67e77f3687c2c811fe75b2defd1cdc3d2afd6aa17R217